docs/generated/UNDERLAY_CONFIG.md

```diff
@@ -590,6 +590,13 @@ The typical use case for this is to optimize cohort breakdown queries on the pri
 
 You can currently specify a maximum of four attributes, because we implement this using BigQuery clustering which has this [limitation](https://cloud.google.com/bigquery/docs/clustered-tables#limitations).
 
+### SZEntity.optimizeSearchByAttributes
```

Comment:
This needs to be a list of lists. Genomics region will have two parameters, the chromosome and position (contig, position).

Reply:
current: variant table (index) includes genomic location, instead of the required contig/position for genomic region. The values are different, hence replacing that field (as suggested by Brian) with contig and position fields.
in PR https://github.com/DataBiosphere/tanagra/pull/1099

Reply:
changed